src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -138,22 +106,31 @@ public static function getFollowing($owner, $page = null)
 		}
 
 		if (empty($page)) {
-			$data['first'] = DI::baseUrl() . '/following/' . $owner['nickname'] . '?page=1';
+			$data['first'] = DI::baseUrl() . $modulePath . $owner['nickname'] . '?page=1';
 		} else {
 			$data['type'] = 'OrderedCollectionPage';
 			$list = [];
 
-			$contacts = DBA::select('contact', ['url'], $condition, ['limit' => [($page - 1) * 100, 100]]);
+			$sql = "SELECT `contact`.`url`
```

Comment:
Shouldn't the condition be indented one more tab?

Reply:
It can, would you like the rest of the SQL query multi-line string to be indented?

Reply:
Since the query string is part of the instruction above, I think that every line of the instruction should be intended, shouldn't it?

Reply:
Here you go.